Yt update

ks00908 opened the pull request:
ADDED
        - youtube support via Lavalink (#4 )
        - Comments in basics.py

FIXED
        - removed jishaku from "initial_extensions" should prevent error on start, it was doubled later in
           code anyway
        - Added check on playing status length when changing via command

CHANGED:
                - Config: added lavalink to config to add node used to play YT links.
                - turned config example from txt to json

REMOVED:
               - dead Events file that was unused

This merge will confirm deployment of feature in official bot

ks00908 committed:
new effectssss

ks00908 committed:
Youtube Update

ADDED
        - youtube support via Lavalink
        - Comments in basics.py

FIXED
        - removed jishaku from "initial_extensions" should prevent error on start
        - Added check on playing status length when changing via command

CHANGED:
                - Config: added lavalink to config to add node used to play YT links.
              - turned config example from txt to json

REMOVED:
               - dead Events file that was unused

This update will be merged with master when it become deployed

ks00908 committed:
Update README.md

pep8speaks commented:
Hello @ks00908! Thanks for updating this PR. We checked the lines you've touched for [PEP 8](https://www.python.org/dev/peps/pep-0008) issues, and found:

* In the file [`doot-doot/cogs/airhorn.py`](https://github.com/ks00908/doot-doot/blob/b3f6097bba61ea27e7003183d74ce552a368778e/doot-doot/cogs/airhorn.py):

> [Line 216:5](https://github.com/ks00908/doot-doot/blob/b3f6097bba61ea27e7003183d74ce552a368778e/doot-doot/cogs/airhorn.py#L216): [E301](https://duckduckgo.com/?q=pep8%20E301) expected 1 blank line, found 0
> [Line 218:23](https://github.com/ks00908/doot-doot/blob/b3f6097bba61ea27e7003183d74ce552a368778e/doot-doot/cogs/airhorn.py#L218): [E211](https://duckduckgo.com/?q=pep8%20E211) whitespace before '('

* In the file [`doot-doot/cogs/basics.py`](https://github.com/ks00908/doot-doot/blob/b3f6097bba61ea27e7003183d74ce552a368778e/doot-doot/cogs/basics.py):

> [Line 60:52](https://github.com/ks00908/doot-doot/blob/b3f6097bba61ea27e7003183d74ce552a368778e/doot-doot/cogs/basics.py#L60): [W291](https://duckduckgo.com/?q=pep8%20W291) trailing whitespace
> [Line 75:80](https://github.com/ks00908/doot-doot/blob/b3f6097bba61ea27e7003183d74ce552a368778e/doot-doot/cogs/basics.py#L75): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (150 > 79 characters)
> [Line 89:45](https://github.com/ks00908/doot-doot/blob/b3f6097bba61ea27e7003183d74ce552a368778e/doot-doot/cogs/basics.py#L89): [W291](https://duckduckgo.com/?q=pep8%20W291) trailing whitespace
> [Line 95:10](https://github.com/ks00908/doot-doot/blob/b3f6097bba61ea27e7003183d74ce552a368778e/doot-doot/cogs/basics.py#L95): [E111](https://duckduckgo.com/?q=pep8%20E111) indentation is not a multiple of four
> [Line 96:14](https://github.com/ks00908/doot-doot/blob/b3f6097bba61ea27e7003183d74ce552a368778e/doot-doot/cogs/basics.py#L96): [E111](https://duckduckgo.com/?q=pep8%20E111) indentation is not a multiple of four
> [Line 96:80](https://github.com/ks00908/doot-doot/blob/b3f6097bba61ea27e7003183d74ce552a368778e/doot-doot/cogs/basics.py#L96): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (83 > 79 characters)
> [Line 97:10](https://github.com/ks00908/doot-doot/blob/b3f6097bba61ea27e7003183d74ce552a368778e/doot-doot/cogs/basics.py#L97): [E111](https://duckduckgo.com/?q=pep8%20E111) indentation is not a multiple of four
> [Line 98:14](https://github.com/ks00908/doot-doot/blob/b3f6097bba61ea27e7003183d74ce552a368778e/doot-doot/cogs/basics.py#L98): [E111](https://duckduckgo.com/?q=pep8%20E111) indentation is not a multiple of four
> [Line 98:80](https://github.com/ks00908/doot-doot/blob/b3f6097bba61ea27e7003183d74ce552a368778e/doot-doot/cogs/basics.py#L98): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (80 > 79 characters)
> [Line 99:14](https://github.com/ks00908/doot-doot/blob/b3f6097bba61ea27e7003183d74ce552a368778e/doot-doot/cogs/basics.py#L99): [E111](https://duckduckgo.com/?q=pep8%20E111) indentation is not a multiple of four
> [Line 99:80](https://github.com/ks00908/doot-doot/blob/b3f6097bba61ea27e7003183d74ce552a368778e/doot-doot/cogs/basics.py#L99): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (81 > 79 characters)

* In the file [`doot-doot/cogs/youtube.py`](https://github.com/ks00908/doot-doot/blob/b3f6097bba61ea27e7003183d74ce552a368778e/doot-doot/cogs/youtube.py):

> [Line 2:80](https://github.com/ks00908/doot-doot/blob/b3f6097bba61ea27e7003183d74ce552a368778e/doot-doot/cogs/youtube.py#L2): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (82 > 79 characters)
> [Line 3:80](https://github.com/ks00908/doot-doot/blob/b3f6097bba61ea27e7003183d74ce552a368778e/doot-doot/cogs/youtube.py#L3): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (142 > 79 characters)
> [Line 15:53](https://github.com/ks00908/doot-doot/blob/b3f6097bba61ea27e7003183d74ce552a368778e/doot-doot/cogs/youtube.py#L15): [W291](https://duckduckgo.com/?q=pep8%20W291) trailing whitespace
> [Line 18:1](https://github.com/ks00908/doot-doot/blob/b3f6097bba61ea27e7003183d74ce552a368778e/doot-doot/cogs/youtube.py#L18): [E302](https://duckduckgo.com/?q=pep8%20E302) expected 2 blank lines, found 1
> [Line 22:1](https://github.com/ks00908/doot-doot/blob/b3f6097bba61ea27e7003183d74ce552a368778e/doot-doot/cogs/youtube.py#L22): [E305](https://duckduckgo.com/?q=pep8%20E305) expected 2 blank lines after class or function definition, found 1
> [Line 25:1](https://github.com/ks00908/doot-doot/blob/b3f6097bba61ea27e7003183d74ce552a368778e/doot-doot/cogs/youtube.py#L25): [E302](https://duckduckgo.com/?q=pep8%20E302) expected 2 blank lines, found 1
> [Line 29:80](https://github.com/ks00908/doot-doot/blob/b3f6097bba61ea27e7003183d74ce552a368778e/doot-doot/cogs/youtube.py#L29): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (105 > 79 characters)
> [Line 31:80](https://github.com/ks00908/doot-doot/blob/b3f6097bba61ea27e7003183d74ce552a368778e/doot-doot/cogs/youtube.py#L31): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (190 > 79 characters)
> [Line 32:80](https://github.com/ks00908/doot-doot/blob/b3f6097bba61ea27e7003183d74ce552a368778e/doot-doot/cogs/youtube.py#L32): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (85 > 79 characters)
> [Line 52:80](https://github.com/ks00908/doot-doot/blob/b3f6097bba61ea27e7003183d74ce552a368778e/doot-doot/cogs/youtube.py#L52): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (82 > 79 characters)
> [Line 55:5](https://github.com/ks00908/doot-doot/blob/b3f6097bba61ea27e7003183d74ce552a368778e/doot-doot/cogs/youtube.py#L55): [E303](https://duckduckgo.com/?q=pep8%20E303) too many blank lines (2)
> [Line 62:80](https://github.com/ks00908/doot-doot/blob/b3f6097bba61ea27e7003183d74ce552a368778e/doot-doot/cogs/youtube.py#L62): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (95 > 79 characters)
> [Line 66:53](https://github.com/ks00908/doot-doot/blob/b3f6097bba61ea27e7003183d74ce552a368778e/doot-doot/cogs/youtube.py#L66): [W291](https://duckduckgo.com/?q=pep8%20W291) trailing whitespace
> [Line 91:80](https://github.com/ks00908/doot-doot/blob/b3f6097bba61ea27e7003183d74ce552a368778e/doot-doot/cogs/youtube.py#L91): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (91 > 79 characters)
> [Line 95:80](https://github.com/ks00908/doot-doot/blob/b3f6097bba61ea27e7003183d74ce552a368778e/doot-doot/cogs/youtube.py#L95): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (85 > 79 characters)
> [Line 105:5](https://github.com/ks00908/doot-doot/blob/b3f6097bba61ea27e7003183d74ce552a368778e/doot-doot/cogs/youtube.py#L105): [E303](https://duckduckgo.com/?q=pep8%20E303) too many blank lines (3)
> [Line 118:80](https://github.com/ks00908/doot-doot/blob/b3f6097bba61ea27e7003183d74ce552a368778e/doot-doot/cogs/youtube.py#L118): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (93 > 79 characters)
> [Line 127:80](https://github.com/ks00908/doot-doot/blob/b3f6097bba61ea27e7003183d74ce552a368778e/doot-doot/cogs/youtube.py#L127): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (83 > 79 characters)
> [Line 133:80](https://github.com/ks00908/doot-doot/blob/b3f6097bba61ea27e7003183d74ce552a368778e/doot-doot/cogs/youtube.py#L133): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (115 > 79 characters)
> [Line 136:29](https://github.com/ks00908/doot-doot/blob/b3f6097bba61ea27e7003183d74ce552a368778e/doot-doot/cogs/youtube.py#L136): [W291](https://duckduckgo.com/?q=pep8%20W291) trailing whitespace
> [Line 142:5](https://github.com/ks00908/doot-doot/blob/b3f6097bba61ea27e7003183d74ce552a368778e/doot-doot/cogs/youtube.py#L142): [E303](https://duckduckgo.com/?q=pep8%20E303) too many blank lines (2)
> [Line 143:80](https://github.com/ks00908/doot-doot/blob/b3f6097bba61ea27e7003183d74ce552a368778e/doot-doot/cogs/youtube.py#L143): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (96 > 79 characters)
> [Line 144:80](https://github.com/ks00908/doot-doot/blob/b3f6097bba61ea27e7003183d74ce552a368778e/doot-doot/cogs/youtube.py#L144): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (95 > 79 characters)
> [Line 147:80](https://github.com/ks00908/doot-doot/blob/b3f6097bba61ea27e7003183d74ce552a368778e/doot-doot/cogs/youtube.py#L147): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (105 > 79 characters)
> [Line 150:80](https://github.com/ks00908/doot-doot/blob/b3f6097bba61ea27e7003183d74ce552a368778e/doot-doot/cogs/youtube.py#L150): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (101 > 79 characters)
> [Line 158:80](https://github.com/ks00908/doot-doot/blob/b3f6097bba61ea27e7003183d74ce552a368778e/doot-doot/cogs/youtube.py#L158): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (89 > 79 characters)
> [Line 159:80](https://github.com/ks00908/doot-doot/blob/b3f6097bba61ea27e7003183d74ce552a368778e/doot-doot/cogs/youtube.py#L159): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (98 > 79 characters)
> [Line 162:80](https://github.com/ks00908/doot-doot/blob/b3f6097bba61ea27e7003183d74ce552a368778e/doot-doot/cogs/youtube.py#L162): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (81 > 79 characters)
> [Line 165:80](https://github.com/ks00908/doot-doot/blob/b3f6097bba61ea27e7003183d74ce552a368778e/doot-doot/cogs/youtube.py#L165): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (88 > 79 characters)
> [Line 169:30](https://github.com/ks00908/doot-doot/blob/b3f6097bba61ea27e7003183d74ce552a368778e/doot-doot/cogs/youtube.py#L169): [W292](https://duckduckgo.com/?q=pep8%20W292) no newline at end of file





##### Comment last updated at 2020-02-29 13:57:10 UTC

ks00908 committed:
HOTFIX

Hotfix to broken char length and attempt at fixing YT cog breaking local sounds

ks00908 committed:
Update youtube.py

ks00908 committed:
Update youtube.py

ks00908 commented:
This pull request got overwritten by rollback commit
